prime-router/src/testIntegration/resources/datatests/FHIR_to_HL7/sample_AOE_1_20230220-0001.fhir

```diff
@@ -13164,8 +13164,17 @@
               }
             ],
             "name" : {
+              "extension" : [
+                {
+                  "url" : "https://reportstream.cdc.gov/fhir/StructureDefinition/xpn2-given-name",
+                  "valueString" : "GENARO"
+                },
+                {
+                  "url" : "https://reportstream.cdc.gov/fhir/StructureDefinition/xpn7-name-type-code",
+                  "valueString" : "L"
+                }
+              ],
               "use" : "official",
-              "text" : "GENARO SURYAN",
```

Comment:
(nit, optional) Could we leave `text` in? Doesn't affect anything, but could be a passive reminder that `name.text` is possible from FHIR senders.